src/examples/hotgym/HelloSPTP.cpp

```diff
@@ -180,7 +197,7 @@ Real64 BenchmarkHotgym::run(UInt EPOCHS, bool useSPlocal, bool useSPglobal, bool
 
       SDR goldTM({COLS});
       const SDR_sparse_t deterministicTM{
-        26, 75 
+        26, 75 //FIXME this is a really bad representation -> improve the params
```

Comment:
for other PR, but the TM has a really awful representation, we need to improve that